docs/en/contributing/packages.md

```diff
@@ -36,7 +36,7 @@ These may include:
 
 All packages use npm and are published to the [npm registry](https://www.npmjs.com/).
 
-### "sideEffects": false
+### Side effects
```

Comment:
The navigation menu look cluttered as a result of code in a title.  This cleans that up and adds clarity in the first sentence.